pages/category/[slug].tsx

```diff
@@ -60,7 +67,11 @@ export const getServerSideProps: GetServerSideProps<Props> = async (context) =>
         };
     }
 
-    const stories = await api.getStoriesFromCategory(category);
+    const page = context.query.page && typeof context.query.page === 'string'
```

Comment:
Is it possible that `page` will be of any other type than `undefined` or `string`? Because to me, the second part of this condition seems redundant

Reply:
From Next types it can be `string | string[] | undefined`.